src/components/eventing/FilterForm.jsx

```diff
@@ -0,0 +1,139 @@
+import React, {useEffect, useState} from "react"
```

Comment:
Don't know for some strange reason the event logs page reloads whenever I open the modal. Please check why this is happening and prevent it.

Reply:
not happening for me

Reply:
It's happening because you are dispatching actions to redux in onChange of CheckBox of filterform. Is there any reason you are dispatching the value of checkbox on change?